openstack/swr/v2/namespaces/requests.go

```diff
@@ -53,3 +53,57 @@ func Delete(c *golangsdk.ServiceClient, id string) (r DeleteResult) {
 	})
 	return
 }
+
+// CreateAccessOptsBuilder allows extensions to add additional parameters to the
+// Create request.
+type CreateAccessOptsBuilder interface {
+	ToAccessCreateMap() (map[string]interface{}, error)
+}
+
+// CreateAccessOpts contains all the values needed to create access of a namespace
+type CreateAccessOpts struct {
+	Users []User
+}
+
+// Access information of a user
+type User struct {
+	// ID of the user
+	UserID string `json:"user_id" required:"true"`
+	// Name of the user
+	UserName string `json:"user_name" required:"true"`
+	// Permission of the user
+	Auth int `json:"auth" required:"true"`
+}
+
+// ToAccessCreateMap builds a create request body from CreateAccessOpts.
+func (opts CreateAccessOpts) ToAccessCreateMap() (map[string]interface{}, error) {
+	return golangsdk.BuildRequestBody(opts, "")
+}
+
+// CreateAccess accepts a CreateAccessOpts struct and uses the values to create access of a namespace.
+func CreateAccess(c *golangsdk.ServiceClient, opts CreateAccessOptsBuilder, nameSpace string) (r CreateAccessResult) {
+	b, err := opts.ToAccessCreateMap()
+	if err != nil {
+		r.Err = err
+		return
+	}
+	reqOpt := &golangsdk.RequestOpts{OkCodes: []int{201}}
+	_, r.Err = c.Post(accessURL(c, nameSpace), b["Users"], &r.Body, reqOpt)
+	return
+}
+
+// Get retrieves the access of a namespace.
+func GetAccess(c *golangsdk.ServiceClient, nameSpace string) (r GetAccessResult) {
+	_, r.Err = c.Get(accessURL(c, nameSpace), &r.Body, &golangsdk.RequestOpts{
+		OkCodes:     []int{200},
```

Comment:
200 is already in the default response codes for GET method.

Reply:
fixed